coursera/commandline.py

```diff
@@ -14,6 +14,14 @@
 from .utils import decode_input
 
 
+def class_required(args):
```

Comment:
Maybe a better name could be "class_name_argument_required", otherwise it's too broad and not immediately clear what is "class".